egs/mgb2/ASR/conformer_ctc/label_smoothing.py

```diff
@@ -0,0 +1,98 @@
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang)
```

Comment:
Could you replace it with a symlink to the corresponding file in librispeech?

Also for the following files:
- subsampling.py
- test_label_smoothing.py
- test_subsampling.py
- test_transformer.py
- transformer.py
- compile_hlg.py
- compute_fbank_musan.py
- convert_transcript_words_to_tokens.py
- download_lm.py
- generate_unique_lexicon.py
- egs/mgb2/ASR/local/prepare_lang.py
- egs/mgb2/ASR/local/prepare_lang_bpe.py
- egs/mgb2/ASR/local/prepare_lexicon.py
- egs/mgb2/ASR/local/test_prepare_lang.py
- egs/mgb2/ASR/local/train_bpe_model.py
- egs/mgb2/ASR/pruned_transducer_stateless5/beam_search.py
- egs/mgb2/ASR/pruned_transducer_stateless5/conformer.py
- egs/mgb2/ASR/pruned_transducer_stateless5/decoder.py
- egs/mgb2/ASR/pruned_transducer_stateless5/encoder_interface.py
- egs/mgb2/ASR/pruned_transducer_stateless5/joiner.py
- egs/mgb2/ASR/pruned_transducer_stateless5/model.py
- egs/mgb2/ASR/pruned_transducer_stateless5/optim.py
- egs/mgb2/ASR/pruned_transducer_stateless5/scaling.py
- egs/mgb2/ASR/pruned_transducer_stateless5/test_model.py
- icefall/diagnostics.py

Reply:
What icefall/diagnostics.py should be linked to?

Reply:
Oh, sorry. You don't neet to create a symlink for `icefall/diagnostics.py`.

Reply:
Done